src/components/modal/report/modal.jsx

```diff
@@ -104,68 +120,104 @@ class ReportModal extends React.Component {
 
                     <Form
                         className="report"
-                        onSubmit={onReport}
+                        onInvalid={this.handleInvalid}
```

Comment:
These "onValid"/"onInvalid" properties are a Formsy thing. Our Form component doesn't have specific support for them, but they seem to work fine just passed through this way. Is this kosher?